feature-libs/cart/import-export/core/model/import-export.model.ts

```diff
@@ -10,3 +10,9 @@ export type FileValidity = {
   allowedExtensions?: string[];
   checkEmptyFile?: Boolean;
 };
+export interface ExportOrderEntry {
+  sku: string;
+  quantity: number | string;
```

Comment:
what do you think about separate model for headers ([key: string]: string) and keeping only`number` here?